<!doctype html>
<html>
<head>
	<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
	<meta charset="utf-8">
	
		<title>Titanium 3.1.1 Android 4.3 and 2.3.3 emulator crash » Community Questions &amp; Answers </title>
	
	<meta name="viewport" content="width=device-width,initial-scale=1">
	<link href="../../related/favicon.png" rel="shortcut icon" type="image/png">
	<link href="//fonts.googleapis.com/css?family=Open+Sans:300,400,500,600,700" rel="stylesheet" type="text/css">
	<link href="//d3ilu1xuwhtfe2.cloudfront.net/e8e0ebd/css/style.css" rel="stylesheet" type="text/css">
	<link href="../../related/qa.css" rel="stylesheet" type="text/css">
	<link href="../../related/hybrid.css" rel="stylesheet" type="text/css">
	
	<link href="../../related/overrides.css" rel="stylesheet" type="text/css">
</head>
<body>

<div class='mast'>    <div class='container'>        <div class='row'>            <div class='col-sm-12'>                <div class='masthead'><h1>Titanium Community Questions & Answer Archive</h1><h2>We felt that 6+ years of knowledge should not die so this is the Titanium Community Questions &amp; Answer Archive</h2></div>            </div>        </div>    </div></div>

<main class="container">
	<div class="row">
		<div class="col-sm-12">
			
		</div>
	</div>
	<div class="row">
		<div class="col-sm-12">
			<article class="question-container">
	<div class="question">
		<header>
			<h1>Titanium 3.1.1 Android 4.3 and 2.3.3 emulator crash</h1>
		</header>
		<section>
			<p>Application type: mobile</p>
<p>Titanium SDK version: 3.1.1 (06&#x2F;15&#x2F;13 16:09 f7592c1)</p>
<p>Platform &amp; version:  Android 4.3 and 2.3.3</p>
<p>Device: Android emulator and Samsung Galaxy S2 with Android 4.0.3</p>
<p>Host Operating System: OSX 10.8.4 Mountain Lion</p>
<p>Titanium Studio: build: 3.1.1.201306112235</p>
<p>Hello guys,</p>
<p>I&#39;m currently having some issues with running my app on the Android emulator and thought maybe someone could help. The problem is that when I launch the app on android emulator (2.3.3 or 4.3) or on actual device (Samsung Galaxy S2) it will crash. </p>
<p>I have to mention that on IPhone simulator won&#39;t crash.  Android 2.3.3 emulator will load the app but then it&#39;s crashing repeateadly (reloads then crashes infintely). On android 4.3 emulator it&#39;s just a black screen when launching the app. From the logs I see that&#39;s something native…and has to do with corrupted memory but there are multiple errors among which I don&#39;t know which is more severe :)</p>
<h3 id="i-tried-to-re-install-android-sdk-and-after-that-the-titanium-studio-it-didn-39-t-change-anything-">I tried to re-install Android SDK and after that the Titanium Studio. It didn&#39;t change anything.</h3>
<p>I attached the logs but they are very huge…sory for that :(</p>
<p>If someone has any ideas on this issue, please help.</p>
<h3 id="thank-you-">Thank you !</h3>
<h2 id="the-logs-for-android-4-3-emulator-">The logs for Android 4.3 emulator:</h2>
<p>`<br>[INFO] Titanium SDK version: 3.1.1 (06&#x2F;15&#x2F;13 16:09 f7592c1)<br>[INFO] Fastdev server running, deploying in Fastdev mode<br>[INFO] Copying CommonJS modules…<br>[INFO] Copying project resources..<br>[INFO] Tiapp.xml unchanged, skipping class generation<br>[INFO] Force including all modules…<br>[INFO] Compiling Javascript Resources …<br>[INFO] Waiting for SDCard to become available..<br>[INFO] Re-launching application … c1rca<br>Error killing app, result: [&#39;App not connected&#39;]<br>[INFO] Launching application … c1rca<br>[INFO] :   Project built successfully in 3s 188ms<br>[INFO] :   Running logcat process: python &quot;&#x2F;Users&#x2F;programm&#x2F;Library&#x2F;Application Support&#x2F;Titanium&#x2F;mobilesdk&#x2F;osx&#x2F;3.1.1.GA&#x2F;android&#x2F;builder.py&quot; &quot;logcat&quot; &quot;&#x2F;Users&#x2F;programm&#x2F;Downloads&#x2F;adt-bundle-mac-x86_64-20130717&#x2F;sdk&quot; &quot;-e&quot;<br>[INFO][PeopleSearchIndexManage(  807)] Rebuilding index…</p>
<h3 id="-error-strictmode-702-a-resource-was-acquired-at-attached-stack-trace-but-never-released-see-java-io-closeable-for-information-on-avoiding-resource-leaks-">[ERROR][StrictMode(  702)] A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.</h3>
<h3 id="-error-strictmode-702-java-lang-throwable-explicit-termination-method-39-close-39-not-called">[ERROR][StrictMode(  702)] java.lang.Throwable: Explicit termination method &#39;close&#39; not called</h3>
<p>[ERROR][StrictMode(  702)]     at dalvik.system.CloseGuard.open(CloseGuard.java:184)<br>[ERROR][StrictMode(  702)]     at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:80)<br>[ERROR][StrictMode(  702)]     at dalvik.system.NativeStart.main(Native Method)<br>[ERROR][GCM     (  702)] Missing token, falling back to GSF</p>
<p>[WARN][Bundle  (  702)] Key GOOG.USER_SERIAL expected String but value was a java.lang.Integer.  The default value &lt;null&gt; was returned.<br>[WARN][Bundle  (  702)] Attempt to cast generated internal exception:<br>[WARN][Bundle  (  702)] java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String<br>android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)<br>[WARN][Bundle  (  702)]     at android.os.HandlerThread.run(HandlerThread.java:61)<br>[INFO][PeopleSearchIndexManage(  807)] Rebuilding index done.<br>[INFO][dalvikvm-heap(  766)] Grow heap (frag case) to 16.615MB for 1286224-byte allocation<br>[INFO][MediaProvider(  669)] Upgrading media database from version 0 to 601, which will destroy all old data</p>
<p>[ERROR][StrictMode( 1114)] A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.<br>[ERROR][StrictMode( 1114)] java.lang.Throwable: Explicit termination method &#39;release&#39; not called</p>
<p>[INFO][GCM     (  807)] Message from Bundle[{error=AUTHENTICATION_FAILED}] null<br>[ERROR][GCM     (  807)] Failed registration AUTHENTICATION_FAILED<br>[WARN][Uploader(  807)] No account for auth token provided<br>[ERROR][StrictMode(  702)] A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.<br>[ERROR][StrictMode(  702)] java.lang.Throwable: Explicit termination method &#39;close&#39; not called</p>
<p>[ERROR][StrictMode(  702)]     at android.os.Looper.loop(Looper.java:137)<br>[ERROR][StrictMode(  702)]     at android.os.HandlerThread.run(HandlerThread.java:61)</p>
<h3 id="-error-sqlitelog-1042-1-no-such-table-account">[ERROR][SQLiteLog( 1042)] (1) no such table: Account</h3>
<p>[WARN][EmailProvider( 1042)] Exception cleaning EmailProvider.dbandroid.database.sqlite.SQLiteException: no such table: Account (code 1): , while compiling: delete from Account WHERE displayName ISNULL;</p>
<p>[ERROR][BluetoothAdapter(  646)] Bluetooth binder is null<br>[INFO][GCoreUlr(  598)] Ensuring that reporting is stopped because of reasons: (no Google accounts)</p>
<p>[INFO][SystemUpdateService(  702)] cancelUpdate (empty URL)<br>[WARN][GCM     (  702)] DIR: &#x2F;data&#x2F;data&#x2F;com.google.android.gms&#x2F;app_APP &#x2F;data&#x2F;data&#x2F;com.google.android.gsf<br>[WARN][dalvikvm( 1275)] PR_CAPBSET_DROP 0 failed: Invalid argument. Please make sure your kernel is compiled with file capabilities support enabled.<br>[WARN][dalvikvm( 1275)] PR_CAPBSET_DROP 1 failed: Invalid argument. Please make sure your kernel is compiled with file capabilities support enabled.</p>
<p>[INFO][ActivityManager(  330)] Start proc com.google.android.apps.maps for service com.google.android.apps.maps&#x2F;com.google.android.apps.gmm.ulr.GcoreUlrService: pid=1275 uid=10026 gids={50026, 3003, 1015, 1028}<br>[INFO][GCoreUlr(  598)] Ensuring that reporting is stopped because of reasons: (no Google accounts)</p>
<h3 id="-error-wvmextractor-40-failed-to-open-libwvm-so">[ERROR][WVMExtractor(   40)] Failed to open libwvm.so</h3>
<h3 id="-error-sqlitedatabase-766-error-inserting-preview_bitmap-b-418dca38-size-168x134-name-widget-com-android-email-x2f-com-android-email-provider-widgetprovider">[ERROR][SQLiteDatabase(  766)] Error inserting preview_bitmap=[B@418dca38 size=168x134 name=Widget:com.android.email&#x2F;com.android.email.provider.WidgetProvider</h3>
<p>[ERROR][SQLiteDatabase(  766)] android.database.sqlite.SQLiteConstraintException: columns name, size are not unique (code 19)<br>[ERROR][SQLiteDatabase(  766)]     at android.database.sqlite.SQLiteConnection.nativeExecuteForLastInsertedRowId(Native Method)</p>
<p>state=CONNECTING<br>[WARN][dalvikvm( 1323)] PR_CAPBSET_DROP 0 failed: Invalid argument. Please make sure your kernel is compiled with file capabilities support enabled.<br>[WARN][dalvikvm( 1323)] PR_CAPBSET_DROP 1 failed: Invalid argument. Please make sure your kernel is compiled with file capabilities support enabled.</p>
<p>[INFO][ActivityManager(  330)] Start proc com.svox.pico for service com.svox.pico&#x2F;.PicoService: pid=1323 uid=10032 gids={50032, 1015, 1028}<br>[INFO][TextToSpeech( 1275)] Sucessfully bound to com.svox.pico<br>[INFO][PackageManager(  330)] Running dexopt on: com.svox.pico<br>[ERROR][GCoreUlr(  598)] Failed to retrieve GMM settings, proceeding without them<br>[ERROR][GCoreUlr(  598)] java.util.concurrent.TimeoutException<br>[ERROR][GCoreUlr(  598)]     at eha.a(SourceFile:80)<br>[ERROR][GCoreUlr(  598)]     at android.os.HandlerThread.run(HandlerThread.java:61)</p>
<p>[WARN][MediaScanner(  669)] Error opening directory &#39;&#x2F;storage&#x2F;sdcard&#x2F;.android_secure&#x2F;&#39;, skipping: Permission denied.<br>[WARN][PeopleReceiver(  807)] Received unexcepted broadcast: android.intent.action.PACKAGE_CHANGED</p>
<p>[INFO][ContactDirectoryManager(  724)] deleted 0 stale rows which don&#39;t have any relevant directory<br>[INFO][ContactDirectoryManager(  724)] Discovered 0 contact directories in 4477ms<br>[INFO][ContactsProvider(  724)] Locale has changed from null to en_US<br>[INFO][ContactsDatabaseHelper(  724)] Switching to locale en_US (ICU v50.1.1.3)</p>
<h3 id="-warn-zipro-660-unable-to-open-zip-39-x2f-data-x2f-local-x2f-tmp-x2f-app-apk-39-no-such-file-or-directory">[WARN][zipro   (  660)] Unable to open zip &#39;&#x2F;data&#x2F;local&#x2F;tmp&#x2F;app.apk&#39;: No such file or directory</h3>
<h3 id="-warn-packageparser-660-unable-to-read-androidmanifest-xml-of-x2f-data-x2f-local-x2f-tmp-x2f-app-apk">[WARN][PackageParser(  660)] Unable to read AndroidManifest.xml of &#x2F;data&#x2F;local&#x2F;tmp&#x2F;app.apk</h3>
<h3 id="-warn-packageparser-660-java-io-filenotfoundexception-androidmanifest-xml">[WARN][PackageParser(  660)] java.io.FileNotFoundException: AndroidManifest.xml</h3>
<p>[WARN][PackageParser(  660)]     at android.content.res.AssetManager.openXmlAssetNative(Native Method)<br>[WARN][PackageParser(  660)]     at dalvik.system.NativeStart.run(Native Method)<br>[WARN][DefContainer(  660)] Failed to parse package<br>[WARN][ActivityManager(  330)] No content provider found for permission revoke: file:&#x2F;&#x2F;&#x2F;data&#x2F;local&#x2F;tmp&#x2F;app.apk</p>
<h2 id="-error-cutils-trace-1388-error-opening-trace-file-no-such-file-or-directory-2-">[ERROR][cutils-trace( 1388)] Error opening trace file: No such file or directory (2)</h2>
<p>[INFO][ActivityManager(  330)] START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.c1rca.c1rca&#x2F;.C1rcaActivity} from pid 1388</p>
<p>[INFO][dalvikvm-heap(  330)] Grow heap (frag case) to 7.185MB for 1536016-byte allocation<br>[INFO][dalvikvm( 1399)] Turning on JNI app bug workarounds for target SDK version 8…</p>
<p>[INFO][ActivityManager(  330)] Start proc com.c1rca.c1rca for activity com.c1rca.c1rca&#x2F;.C1rcaActivity: pid=1399 uid=10050 gids={50050, 1015, 3003, 1028}<br>[INFO][ARMAssembler(   37)] generated scanline__00000077:03515104_00009001_00000000 [131 ipp] (153 ins) at [0x4191d1f0:0x4191d454] in 9293002 ns</p>
<p>[ERROR][TiApplication( 1399)] (KrollRuntimeThread) [912,966] APP PROXY: ti.modules.titanium.app.AppModule@417a98d8<br>[WARN][V8Object( 1399)] Runtime disposed, cannot set property &#39;userAgent&#39;<br>[INFO][ActivityManager(  330)] START u0 {cmp=com.c1rca.c1rca&#x2F;org.appcelerator.titanium.TiActivity (has extras)} from pid 1399<br>[INFO][TiAPI   ( 1399)]  Stored login</p>
<p>[INFO][dalvikvm-heap( 1399)] Grow heap (frag case) to 4.723MB for 1536016-byte allocation<br>[INFO][dalvikvm-heap( 1399)] Grow heap (frag case) to 5.629MB for 635812-byte allocation<br>[WARN][TiUIActivityIndicator( 1399)] (main) [234,1341] Invalid value &quot;0&quot; for style.</p>
<h2 id="-error-dalvikvm-1399-jni-error-app-bug-local-reference-table-overflow-max-512-">[ERROR][dalvikvm( 1399)] JNI ERROR (app bug): local reference table overflow (max=512)</h2>
<p>[WARN][dalvikvm( 1399)] JNI local reference table (0x2a22cfa0) dump:<br>[WARN][dalvikvm( 1399)]   Last 10 entries (of 512):<br>[WARN][dalvikvm( 1399)]       511: 0x4196b738 org.appcelerator.kroll.KrollDict<br>[WARN][dalvikvm( 1399)]       510: 0x417b11c8 org.appcelerator.titanium.proxy.ActivityProxy<br>[WARN][dalvikvm( 1399)]       509: 0x4178faa0 ti.modules.titanium.ui.TabGroupProxy<br>[WARN][dalvikvm( 1399)]       508: 0x41a9bfc8 org.appcelerator.titanium.proxy.ActionBarProxy<br>[WARN][dalvikvm( 1399)]       507: 0x41791940 ti.modules.titanium.ui.TabProxy<br>[WARN][dalvikvm( 1399)]       506: 0x417b11c8 org.appcelerator.titanium.proxy.ActivityProxy</p>
<p>[WARN][dalvikvm( 1399)]   Summary:<br>[WARN][dalvikvm( 1399)]         1 of java.lang.String<br>[WARN][dalvikvm( 1399)]         1 of java.util.HashMap$KeySet<br>[WARN][dalvikvm( 1399)]         2 of org.appcelerator.kroll.runtime.v8.V8Object (2 unique instances)<br>[WARN][dalvikvm( 1399)]       127 of org.appcelerator.titanium.proxy.ActivityProxy (1 unique instances)<br>[WARN][dalvikvm( 1399)]         2 of org.appcelerator.kroll.KrollDict (2 unique instances)<br>[WARN][dalvikvm( 1399)]       127 of ti.modules.titanium.ui.TabGroupProxy (1 unique instances)<br>[WARN][dalvikvm( 1399)]       127 of ti.modules.titanium.ui.TabProxy (1 unique instances)<br>[WARN][dalvikvm( 1399)]       125 of org.appcelerator.titanium.proxy.ActionBarProxy (1 unique instances)</p>
<p>###[ERROR][dalvikvm( 1399)] Failed adding to JNI local ref table (has 512 entries)</p>
<p>###[ERROR][dalvikvm( 1399)] VM aborting</p>
<h2 id="f-x2f-libc-1399-fatal-signal-11-sigsegv-at-0xdeadd00d-code-1-thread-1417-krollruntimethr-">F&#x2F;libc    ( 1399): Fatal signal 11 (SIGSEGV) at 0xdeadd00d (code=1), thread 1417 (KrollRuntimeThr)</h2>
<p>[INFO][DEBUG   (   35)] <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong><br>[INFO][DEBUG   (   35)] Build fingerprint: &#39;generic&#x2F;google_sdk&#x2F;generic:4.3&#x2F;JB_MR2&#x2F;774058:eng&#x2F;test-keys&#39;<br>[INFO][DEBUG   (   35)] Revision: &#39;0&#39;<br>[INFO][DEBUG   (   35)] pid: 1399, tid: 1417, name: UNKNOWN  &gt;&gt;&gt; com.c1rca.c1rca &lt;&lt;&lt;<br>[INFO][DEBUG   (   35)] signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr deadd00d</p>
<p>[INFO][WindowState(  330)] WIN DEATH: Window{41765ab0 u0 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity}<br>[INFO][WindowManager(  330)] Screenshot max retries 4 of Token{41752648 ActivityRecord{417637b0 u0 com.c1rca.c1rca&#x2F;org.appcelerator.titanium.TiActivity}} appWin=Window{41762950 u0 Starting com.c1rca.c1rca} drawState=4</p>
<p>android.app.ApplicationThreadProxy.schedulePauseActivity(ApplicationThreadNative.java:635)<br>[WARN][ActivityManager(  330)]     at com.android.server.am.ActivityStack.startPausingLocked(ActivityStack.java:990)</p>
<p>[WARN][ActivityManager(  330)]     at com.android.server.am.ActivityManagerService.handleAppCrashLocked(ActivityManagerService.java:834</p>
<p>com.android.server.am.ActivityManagerService.handleApplicationCrashInner(ActivityManagerService.java<br>[INFO][ActivityManager(  330)] Start proc com.c1rca.c1rca for activity com.c1rca.c1rca&#x2F;.C1rcaActivity: pid=1428 uid=10050 gids={50050, 1015, 3003, 1028}<br>[INFO][ActivityManager(  330)] Process com.c1rca.c1rca (pid 1399) has died and restarted (pid 1428).</p>
<p>###[ERROR][NativeCrashListener(  330)] Unable to report native crash</p>
<p>###[ERROR][NativeCrashListener(  330)] java.lang.IndexOutOfBoundsException: Invalid index 1, size is 1<br>[ERROR][NativeCrashListener(  330)]     at java.util.ArrayList.throwIndexOutOfBoundsException(ArrayList.java:255)<br>[ERROR][NativeCrashListener(  330)]     at java.util.ArrayList.get(ArrayList.java:308)<br>[ERROR][NativeCrashListener(  330)]     at com.android.server.am.NativeCrashListener$NativeCrashReporter.run(NativeCrashListener.java:86)<br>`</p>
<h2 id="the-logs-for-android-2-3-3-emulator-">The logs for Android 2.3.3 emulator:</h2>
<p>`<br>[INFO] Titanium SDK version: 3.1.1 (06&#x2F;15&#x2F;13 16:09 f7592c1)</p>
<p>[WARN][InputManagerService(   62)] Got RemoteException sending setActive(false) notification to pid 618 uid 10038<br>[INFO][ActivityManager(   62)] Displayed com.c1rca.c1rca&#x2F;.C1rcaActivity: +4m47s976ms (total +3s554ms)<br>[INFO][ActivityManager(   62)] Displayed com.c1rca.c1rca&#x2F;org.appcelerator.titanium.TiActivity: +1s588ms<br>[INFO][EventLogService(  194)] Aggregate from 1379503037486 (log), 1379503037486 (data)</p>
<p>###[WARN][dalvikvm(  634)] ReferenceTable overflow (max=512)<br>[WARN][dalvikvm(  634)] Last 10 entries in JNI local reference table:<br>[WARN][dalvikvm(  634)]   502: 0x406773d0 cls=Lti&#x2F;modules&#x2F;titanium&#x2F;ui&#x2F;TabGroupProxy; (164 bytes)<br>[WARN][dalvikvm(  634)]   503: 0x40680028 cls=Lorg&#x2F;appcelerator&#x2F;titanium&#x2F;proxy&#x2F;ActivityProxy; (92 bytes)<br>[WARN][dalvikvm(  634)]   504: 0x40679208 cls=Lti&#x2F;modules&#x2F;titanium&#x2F;ui&#x2F;TabProxy; (116 bytes)<br>[WARN][dalvikvm(  634)]   505: 0x406773d0 cls=Lti&#x2F;modules&#x2F;titanium&#x2F;ui&#x2F;TabGroupProxy; (164 bytes)</p>
<p>###[ERROR][dalvikvm(  634)] Failed adding to JNI local ref table (has 512 entries)<br>[INFO][dalvikvm(  634)] &quot;KrollRuntimeThread&quot; prio=5 tid=10 RUNNABLE<br>[INFO][dalvikvm(  634)]   | group=&quot;main&quot; sCount=0 dsCount=0 obj=0x40627d00 self=0x2cea70<br>[INFO][dalvikvm(  634)]   | sysTid=644 nice=0 sched=0&#x2F;0 cgrp=default handle=2947344</p>
<p>###[ERROR][dalvikvm(  634)] VM aborting<br>[INFO][DEBUG   (   31)] <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong> <strong><em> </em></strong><br>[INFO][DEBUG   (   31)] Build fingerprint: &#39;generic&#x2F;google_sdk&#x2F;generic:2.3.4&#x2F;GINGERBREAD&#x2F;123630:eng&#x2F;test-keys&#39;<br>[INFO][DEBUG   (   31)] pid: 634, tid: 644  &gt;&gt;&gt; com.c1rca.c1rca &lt;&lt;&lt;</p>
<h2 id="-info-debug-31-signal-11-sigsegv-code-1-segv_maperr-fault-addr-deadd00d">[INFO][DEBUG   (   31)] signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr deadd00d</h2>
<p>[INFO][DEBUG   (   31)]  r0 fffffec4  r1 deadd00d  r2 00000026  r3 00000000<br>[INFO][DEBUG   (   31)]  r4 800a45c0  r5 40013be0  r6 002ceba8  r7 41924f9c<br>[INFO][DEBUG   (   31)]  r8 80d67478  r9 002f7dec  10 40673a08  fp 002f7de8<br>[INFO][DEBUG   (   31)]  ip 800a4720  sp 4431ec28  lr afd19375  pc 80045a4a  cpsr 20000030<br>[INFO][DEBUG   (   31)]          #00  pc 00045a4a  &#x2F;system&#x2F;lib&#x2F;libdvm.so<br>[INFO][DEBUG   (   31)]          #01  pc 00049e06  &#x2F;system&#x2F;lib&#x2F;libdvm.so</p>
<p>[ERROR][InputDispatcher(   62)] channel &#39;407a9e78 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity (server)&#39; ~ Consumer closed input channel or an error occurred.  events=0x8</p>
<p>###[ERROR][InputDispatcher(   62)] channel &#39;407a9e78 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity (server)&#39; ~ Channel is unrecoverably broken and will be disposed!<br>[INFO][ActivityManager(   62)] Process com.c1rca.c1rca (pid 634) has died.<br>[INFO][WindowManager(   62)] WIN DEATH: Window{407a9e78 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity paused=false}</p>
<p>###[ERROR][TiApplication(  651)] (KrollRuntimeThread) [514,514] APP PROXY: ti.modules.titanium.app.AppModule@406425c0<br>[WARN][V8Object(  651)] Runtime disposed, cannot set property &#39;userAgent&#39;<br>[INFO][dalvikvm(  651)] Could not find method org.appcelerator.titanium.TiBaseActivity.getActionBar, referenced from method ti.modules.titanium.ui.TabGroupProxy.windowCreated<br>[WARN][dalvikvm(  651)] VFY: unable to resolve virtual method 7642: Lorg&#x2F;appcelerator&#x2F;titanium&#x2F;TiBaseActivity;.getActionBar ()Landroid&#x2F;app&#x2F;ActionBar;<br>[INFO][ActivityManager(   62)] Starting: Intent { cmp=com.c1rca.c1rca&#x2F;org.appcelerator.titanium.TiActivity (has extras) } from pid 651<br>[INFO][TiAPI   (  651)]  Stored login<br>[INFO][TiAPI   (  651)]  storedUsername:null<br>[INFO][TiAPI   (  651)]  storedPwdHash:null<br>[INFO][TiAPI   (  651)]  Ti.App.FriendName: 0</p>
<p>[WARN][dalvikvm(  651)] ReferenceTable overflow (max=512)<br>[WARN][dalvikvm(  651)] Last 10 entries in JNI local reference table:<br>[WARN][dalvikvm(  651)]   502: 0x406778e0 cls=Lti&#x2F;modules&#x2F;titanium&#x2F;ui&#x2F;TabGroupProxy; (164 bytes)</p>
<p>[WARN][dalvikvm(  651)] JNI local reference table summary (512 entries):</p>
<p>###[[ERROR][dalvikvm(  651)] Failed adding to JNI local ref table (has 512 entries)<br>[INFO][dalvikvm(  651)] &quot;KrollRuntimeThread&quot; prio=5 tid=10 RUNNABLE<br>org.appcelerator.kroll.KrollRuntime$KrollRuntimeThread.run(KrollRuntime.java:112)<br>[INFO][dalvikvm(  651)]<br>[ERROR][dalvikvm(  651)] VM aborting<br>[INFO][DEBUG   (   31)] Build fingerprint: &#39;generic&#x2F;google_sdk&#x2F;generic:2.3.4&#x2F;GINGERBREAD&#x2F;123630:eng&#x2F;test-keys&#39;<br>[INFO][DEBUG   (   31)] pid: 651, tid: 661  &gt;&gt;&gt; com.c1rca.c1rca &lt;&lt;&lt;<br>[###[INFO][DEBUG   (   31)] signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr deadd00d<br>[INFO][DEBUG   (   31)]  r0 fffffec4  r1 deadd00d  r2 00000026  r3 00000000</p>
<p>[INFO][BootReceiver(   62)] Copying &#x2F;data&#x2F;tombstones&#x2F;tombstone_06 to DropBox (SYSTEM_TOMBSTONE)<br>[ERROR][InputDispatcher(   62)] channel &#39;4072fe10 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity (server)&#39; ~ Consumer closed input channel or an error occurred.  events=0x8<br>[ERROR][InputDispatcher(   62)] channel &#39;4072fe10 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity (server)&#39; ~ Channel is unrecoverably broken and will be disposed!<br>[INFO][ActivityManager(   62)] Process com.c1rca.c1rca (pid 651) has died.<br>[INFO][WindowManager(   62)] WIN DEATH: Window{4072fe10 com.c1rca.c1rca&#x2F;com.c1rca.c1rca.C1rcaActivity paused=false}<br>[INFO][ActivityManager(   62)] Start proc com.c1rca.c1rca for activity com.c1rca.c1rca&#x2F;.C1rcaActivity: pid=667 uid=10038 gids={1015, 3003}<br>[INFO][dalvikvm(  667)] Could not find method android.app.Application.onTrimMemory, referenced from method org.appcelerator.titanium.TiApplication.onTrimMemory<br>[WARN][dalvikvm(  667)] VFY: unable to resolve virtual method 123: Landroid&#x2F;app&#x2F;Application;.onTrimMemory (I)V<br>`</p>

		</section>
		<footer>
			<div class="author">
				&mdash; asked <span title="September 18th 2013, 11:57:57 am">September 18th 2013</span>
				by <span class='authorname'>Mihai Marginean</span>
			</div>

			
				<ul class="tags">
					
						<li><span class='tagname'>android</span></li>
					
						<li><span class='tagname'>crash</span></li>
					
						<li><span class='tagname'>emulator</span></li>
					
						<li><span class='tagname'>memory</span></li>
					
						<li><span class='tagname'>titanium</span></li>
					
				</ul>
			

			<section>
				<h5>0 Comments</h5>
				
			</section>
		</footer>
		<aside class="vote-box">
			<div class="score"><span>0</span> Votes</div>
			<div class="answers"><span>2</span> Answers</div>
		</aside>
	</div>

	<hr>

	<h3>2 Answers</h3>

	
		<ul class="answers">
			
				<li class="answer">
				
					<article id="answer-268774">
				
						<section>
							<p>Hi <strong>Mihai Marginean</strong></p>
<p>I have already answered this type of question  <a href="http:&#x2F;&#x2F;developer.appcelerator.com&#x2F;question&#x2F;157317&#x2F;unable-to-launch-the-application-in-android-emulator">this link</a> you can follow that installetion process to avoid this problem </p>
<p>Thanks,</p>

						</section>
						<footer>
							<div class="author">
								&mdash; answered <span title="September 19th 2013, 5:29:47 am">September 19th 2013</span>
								by <span class='authorname'>Motiur Rahman</span><br>
								<a class="icon-bg icon-link" href="../../question/157314/titanium-311-android-43-and-233-emulator-crash.html#answer-268774" rel="permalink">permalink</a>
							</div>

							<h5>1 Comment</h5>
							
								<ul class="comments">
								
									<li class="comment">
										<p><p>Hi Motiur Rahman,</p>
<p>Thank you for your answer. I did everything you mentioned in that post but unfortunately it doesn&#39;t resolve the problem…still happening the same stuff even after un-installing and installing everything in that particular order.</p>
<p>Thanks,</p>
</p>
										<div class="author">
											&mdash; commented <span title="September 23rd 2013, 2:36:21 pm">September 23rd 2013</span>
											by <span class='authorname'>Mihai Marginean</span>
										</div>
									</li>
								
								</ul>
							
						</footer>
						<aside class="vote-box">
							<div class="score"><span>0</span> Votes</div>
						</aside>
					</article>
				</li>
			
				<li class="answer">
				
					<article id="answer-269060">
				
						<section>
							<p>The problem was caused by trying to log a <a href="http:&#x2F;&#x2F;stackoverflow.com&#x2F;questions&#x2F;13861254&#x2F;json-stringify-deep-objects">deep object</a> with JSON.stringify. As soon as I commented that line the app start working fine.</p>

						</section>
						<footer>
							<div class="author">
								&mdash; answered <span title="September 24th 2013, 10:36:50 am">September 24th 2013</span>
								by <span class='authorname'>Mihai Marginean</span><br>
								<a class="icon-bg icon-link" href="../../question/157314/titanium-311-android-43-and-233-emulator-crash.html#answer-269060" rel="permalink">permalink</a>
							</div>

							<h5>1 Comment</h5>
							
								<ul class="comments">
								
									<li class="comment">
										<p><p>I had this issue too.  It&#39;s a pity because I use JSON.stringify a lot for logging and debugging.</p>
</p>
										<div class="author">
											&mdash; commented <span title="October 31st 2013, 1:25:41 pm">October 31st 2013</span>
											by <span class='authorname'>Peter Evans</span>
										</div>
									</li>
								
								</ul>
							
						</footer>
						<aside class="vote-box">
							<div class="score"><span>0</span> Votes</div>
						</aside>
					</article>
				</li>
			
		</ul>
	
</article>

		</div>
		<div class="col-sm-3"></div>
	</div>
	<div class="row"><div class="signoff">The ownership of individual contributions to this community generated content is retained by the authors of their contributions.<br>All trademarks remain the property of the respective owner.</div></div>
</main>


</body>
</html>
